### PR TITLE
Remove ui only documentation

### PR DESCRIPTION
**Pull request by kevinlinxc**
Since the removal of ui only was merged (https://github.com/rubygems/rubygems.org/pull/3084), the guides should be updated. 

I remove other mentions of it and only talk about it in the place where dropdown is mentioned, explaining that it will still show up for people that have it as their set MFA level. Perhaps this bit is unnecessary but it feels like harmless transparency.

Another idea that I think we should do is link the ui only removal blog post.

### CODE REVIEW

File: setting-up-multifactor-authentication.md
Comment (kevinlinxc) [inline setting-up-multifactor-authentication.md:58]: I imagine the blog post as being linked on `removed`

**Comment by kevinlinxc** [comment]
image

**Review by jenshenny — APPROVED** [review]
LGTM, thanks @kevinlinxc

File: setting-up-multifactor-authentication.md
Comment (jenshenny) [reply setting-up-multifactor-authentication.md]: Sure, it would need to be created in the blog repo here: https://github.com/rubygems/rubygems.github.io. Would you like to write up a draft since you worked on this?

Clarifying on my original question, do you have a preference on the merge order? ie. merging this before the blog post has been posted and adding the link after or waiting until the blog has been merged and then merging this?

Comment (kevinlinxc) [reply setting-up-multifactor-authentication.md]: Sorry, I undercommunicated. I meant that I think the blog post linking could be a new pull request after this PR is merged. As for writing the blog post, I could do it, but I don't yet understand why UI only is insecure. Is it just that the CLI usage is more common?

**Comment by kevinlinxc** [comment]
Ok, we wrote the blog post, so we might as well wait for it and link it.
So, this PR is blocked by https://github.com/rubygems/rubygems.github.io/pull/119

**Comment by duckinator** [comment]
The PR @kevinlinxc mentioned [was published in August 2022](https://blog.rubygems.org/2022/08/22/mfa-ui-only-removed.html). Is there a reason this wasn't merged?

**Comment by duckinator** [comment]
I rebased this off main and merged it, over at #385.

**Comment by duckinator** [comment]
Thanks for the help, @kevinlinxc, even if it took a while to get merged. :slightly_smiling_face:

**Comment by kevinlinxc** [comment]
> Thanks for the help, @kevinlinxc, even if it took a while to get merged. 🙂

Thanks for carrying it across the finish line @duckinator !

**Pull request closed without merging**
